### PR TITLE
feat: install prompts and refactor init.js

### LINKED ISSUE #156: Migrate to enquirer
## Present Scenario
We're currently using [inquirer.js](https://github.com/SBoudrias/Inquirer.js/) for the interactive prompts. This has led to a huge rise in the bundle size as inquirer depends on a couple of packages.

## Proposed Solution
 Switch to [enquirer](https://github.com/enquirer/enquirer).

### PR DESCRIPTION
**Pull request by aquibbaig**
**What kind of change does this PR introduce?**
- Switch from inquirer to prompts
- refactor init.js

**Did you add tests for your changes?**
Not yet

**If relevant, did you update the documentation?**
No

Fixes #156

### CODE REVIEW

**Comment by aquibbaig** [comment]
That old commit was disordered to track, review and make changes, so made a new one instead.

**Comment by aquibbaig** [comment]
@jamesgeorge007 We would most likely squash all the commits before merge!

File: src/commands/basic/add.js [hunk 1]
```diff
@@ -71,16 +71,20 @@ const addPlugins = async () => {
...
+    type: 'multiselect',
+    name: 'pluginChoices',
     message: 'Select the plugins to install',
     choices: installablePlugins,
     default: [
       installablePlugins[Math.floor(Math.random() * installablePlugins.length)],
     ],
   });
 
+  // The choice's index in the array will be used as its value if it is not specified.
```
Comment (jamesgeorge007) [inline src/commands/basic/add.js]:
```suggestion
  // The index of choice in the array will be used as its value if not specified.
```

**Comment by aquibbaig** [comment]
@jamesgeorge007 Feel free to drop a review and resolve the conflicts.

**Comment by aquibbaig** [comment]
Have to redo the changes here, I mistook the default branch to be `master`, have to make these changes on top of `develop`. Is that right @jamesgeorge007 ?

**Comment by jamesgeorge007** [comment]
Exactly.

**Comment by aquibbaig** [comment]
@jamesgeorge007 I updated the branch on top of `develop`. There seems to be some issues with `package-lock.json`. Can you delete the file and do a fresh `npm install`.

**Pull request closed without merging**